purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/PaywallColor.kt

```diff
@@ -10,6 +10,10 @@ import kotlinx.serialization.descriptors.PrimitiveSerialDescriptor
 import kotlinx.serialization.descriptors.SerialDescriptor
 import kotlinx.serialization.encoding.Decoder
 import kotlinx.serialization.encoding.Encoder
+import java.util.Locale
+
+private const val COLOR_HEX_FORMAT = "#%06X"
+private const val COLOR_WHITE_HEX = 0xFFFFFF
```

Comment:
Note that iOS supports `RGBA` too. Even though the dashboard doesn't support it yet, it might be nice to allow it in the SDKs so that they can work when they add them there.

Reply:
https://linear.app/revenuecat/issue/PWL-269/support-rgba